_rules/element-marked-decorative-is-not-exposed-46ca7f.md

```diff
@@ -77,15 +77,15 @@ This `nav` element is [marked as decorative][] through its `role` attribute and
 This `img` element is [marked as decorative][] through its empty `alt` attribute and has [semantic role][] of `none`.
 
 ```html
-<img src="test-assets/shared/w3c-logo.png" alt="" />
+<img src="/test-assets/shared/w3c-logo.png" alt="" />
 ```
 
 #### Passed Example 5
 
 This `img` element is [marked as decorative][] through its `role` attribute and has a [semantic role][] of `none` because own attributes are not required to be exposed and thus do no trigger the [presentational roles conflict resolution][].
```

Comment:
```suggestion
This `img` element is [marked as decorative][] through its `role` attribute and has a [semantic role][] of `none` because own attributes are not required to be exposed and thus do not trigger the [presentational roles conflict resolution][].
```